core/src/main/java/com/amplifyframework/predictions/models/LanguageType.java

```diff
@@ -0,0 +1,647 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amplifyframework.predictions.models;
+
+/**
+ * Enum of different types of language.
+ */
+@SuppressWarnings("JavadocVariable")
+public enum LanguageType {
```

Comment:
Can we just use `java.util.Locale`?

Do we actually support all of the things in this enum?

If we actually must use this enum, can we use the `SCREAMING_SNAKE("abbrev")` format?

If we must use this list, can we sort it?

If we must use this list, can we provide a `fromLocale(...)` helper, which finds the correct Amazon-defined language-type, from something more familiar to the user?

Reply:
I... gave up 1/3 of the way into doing exactly what you mentioned here manually... Not sure about using `Locale` though.

Reply:
Yea, I agree we don't want to do it manually. But, we also don't want to maintain this list. Either way, it would be good to have the platform / Java / a computer program generate/maintain this listing for us.